src/Agent.cs

```diff
@@ -25,6 +25,11 @@ namespace Instrumental
   public class Agent
   {
     /// <summary>
+    ///   The agent version, according to the agent.  See script/check_version for things that must match this
```

Comment:
This is a public repo, so, it's not a secret or anything.

On the other hand, you are right, check version should point down, this does not need to point up.